model-generator-commons/src/main/java/io/ballerina/modelgenerator/commons/CommonUtils.java

```diff
@@ -764,4 +764,9 @@ public static boolean isHttpModule(Symbol symbol) {
         ModuleID moduleId = module.get().id();
         return moduleId.orgName().equals(CommonUtils.BALLERINA_ORG_NAME) && moduleId.packageName().equals("http");
     }
+
+    public static String getClassType(String packageName, String clientName) {
+        String importPrefix = packageName.substring(packageName.lastIndexOf('.') + 1);
+        return String.format("%s:%s", importPrefix, clientName);
+    }
```

Comment:
If we are putting this to common utils, IMO, let's use `typeName` instead of `clientName`